cosmo_tester/resources/terraform/openstack-linux-cli-test.tf

```diff
@@ -159,10 +161,17 @@ resource "openstack_compute_instance_v2" "manager_server" {
     agent = "false"
   }
 
+  provisioner "file" {
+    source = "${var.private_key_path}"
+    destination = "/tmp/key.pem"
+  }
+
   provisioner "remote-exec" {
     inline = [
-      "echo hello world",
-      "sudo yum update openssl -y"
+      "echo Setting permissions for private key file: ${var.remote_key_path}",
+      "sudo cp /tmp/key.pem ${var.remote_key_path}",
+      "sudo chown cfyuser:cfyuser ${var.remote_key_path}",
```

Comment:
<minor>
I tend to prefer:
sudo chown cfyuser: ${file_path}

This changes the user to the user specified and the group to that user's default group.